team-agreements/definition-of-ready/readme.md

```diff
@@ -1,8 +1,30 @@
-# Team Definition Of Ready Agreement
+# Definition of Ready
 
-## Content TODO's
+At the time the development team picks a user story from the top of the backlog to estimate during the Sprint planning, they consider that the user story has a certain level of detail that helps them to give an accurate estimate of completion of the user story within the sprint. This level of information is what makes the story to be Ready to be developed in the sprint.
 
-- Define criteria for a story to be ready to pull into a sprint.
-  - Examples
-    - Has estimate
-    - Team understands implementation/design
+  `In case a user story is not Ready in the beginning of the Sprint it increases the chance that your story will not be done at the end of this print.`
+
+## What it is
+
+Definition of Ready is the agreement made by the scrum team around how complete a user story should be to be selected as candidate for estimation in the sprint planning.
+
+It can be understood as a checklist that helps the Product Owner to ensure that the user story he wrote contains all the necessary and the enough content to the scrum team.
+
+### Examples of ready checklist items:
+
+- The description had enough details and input values mandatory in the implementation.

```

Comment:
Suggestion for the checklist items (for consistency with other checklist items in the playbook)

- [  ] Does the description have the details and input values required to implement the story?
- [  ] Does the user story have clear and complete acceptance criteria?
- [  ] Does the user story address the business need?
- [  ] Can we measure the acceptance criteria?

Reply:
done. Thanks, @TessFerrandez